### ISSUE TITLE
[Keyboard Navigation - Help - JavaScript]: Focus does not moves to the 'JavaScript' pane after selecting java script under help. 

### ISSUE THREAD

**Issue by v-swasha** [Microsoft/pxt#2527]
**User Experience:**
Users who are dependent on keyboard navigation or people with mobility impairments who cannot use a mouse will not be able to understand and interact with the page properly if focus does not moves to desired element on the page.

**Test Environment:**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.

**Repro Steps**
1-Navigate to https://makecode.microbit.org/acc
2-Navigate to the Micro bit section element and select Code control given on it.
3-Navigate to the Help control lying in the header section on the page and select it.
4-Navigate to 'JavaScript' control and select it.
5-Verify that focus moves to the java script pane or not after selecting JavaScript under help.

**Actual Result:**
Focus does not moves to the 'JavaScript' pane after selecting java script under help. After selecting JavaScript under help JavaScript pane appears but focus moving to settings button.

**Expected Result:**
Focus should moves to the 'JavaScript' pane after selecting java script under help.

**Note:**
This issue exists with All the menu option showing under Help Drop down like 'Project,Refrence,Block,Hardware,Buy'

**MAS Reference:**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a4bbd37d-ee11-485e-b5c8-0539f0ac94b2}

**Suggested fix:** 
Tips: 
1-All the changes in context should be predictable by user and be triggered by user interaction only 
2-Moving focus should not trigger change in context 
e.g. 
Page should trigger a window/dialog or any expanding menu only when user does the action (SPACEBAR/ENTER) 

**Key References for Fix:** 
https://www.w3.org/TR/WCAG20-TECHS/G107.html  

**Please refer the attachment**
[MAS3.2.1_JavaScript_Focus not moving to java script pane after select java script.zip](https://github.com/Microsoft/pxt/files/1176281/MAS3.2.1_JavaScript_Focus.not.moving.to.java.script.pane.after.select.java.script.zip)

**Comment by v-amik** [comment]
Review done

**Comment by v-mesala** [comment]
Verified and URL: https://makecode.microbit.org/acc and issue is fixed. Hence closing the bug
